src/tests/base.py

```diff
@@ -2,11 +2,12 @@
 import os
 from os.path import join
 from subprocess import check_output
-from unittest import TestCase
+# pylint: disable-msg=import-error
+from unittest2 import TestCase
```

Comment:
got confused by this, but the most consistent choice seems to use `unittest2` (the backport of unittest for Python 2) for all versions, so that the methods like `assertRegex` are always perfectly the same

Reply:
yeah, unittest2 was a little old and I think it missed assertCountEquals that I had to monkey patch, but it fixed a host of other smaller problems